hw7_Zhurylo/js/index.js

```diff
@@ -0,0 +1,11 @@
+document.addEventListener('DOMContentLoaded', function () {
+    const firstNumber = +prompt('Enter the first number')
+    const secondNumber = +prompt('Enter the second number')
+    const thirdNumber = +prompt('Enter the third number')
+    if ( (isNaN(firstNumber)) || (isNaN(secondNumber)) || (isNaN(thirdNumber))) {
+        alert('One of the values you entered is not a number! You need to enter the numbers')
+        location. reload()
```

Comment:
Looks like this is an error. 
```suggestion
        location.reload()
```